pkgs/development/tools/hover/default.nix

```diff
@@ -0,0 +1,70 @@
+{ buildGoModule, fetchFromGitHub, fetchurl, glfw, libGL, libX11, libXcursor, libXi,
+  libXinerama, libXrandr, libXxf86vm, makeWrapper, pkgconfig, stdenv, unzip}:
+
+buildGoModule rec {
+  pname = "hover";
+  version = "unstable-2020-01-23";
+
+  subPackages = [ "." ];
+
+  modSha256 = "019lg3s86965jyqfy83hb52y5rb6g91prsg3ckjxbcnca68lsdja";
+
+  src = fetchFromGitHub {
+    rev = "a6f11fe3c4946c9e7cce75360902bd79a077ccf4";
+    owner = "go-flutter-desktop";
+    repo = pname;
+    sha256 = "0pwdzd34zcbyspl7qiwfcrppr6zxp2zliqg9n8s6s3l6k7mffl9j";
+  };
+
+  libFlutter = fetchurl {
+    name = "libflutter_engine.zip";
+    url = if stdenv.isDarwin then "https://storage.googleapis.com/flutter_infra/flutter/051762798888ebea4666c5f9c9f4c0e4d3b3b481/darwin-x64/FlutterEmbedder.framework.zip"
+          else "https://storage.googleapis.com/flutter_infra/flutter/051762798888ebea4666c5f9c9f4c0e4d3b3b481/linux-x64/linux-x64-embedder";
+    sha256 = "15p6cdvi6rajjs6vv79y7vsrwwpg2kk9qlg14891wrmdn9nvpqb4";
+  };
+
+  nativeBuildInputs = [ pkgconfig unzip makeWrapper ];
+
+  buildInputs = [
+    glfw
+    libX11
+    libXcursor
+    libXinerama
+    libXrandr
+    libXxf86vm
+    libXi
+    libGL
+  ];
+
+  patches = [
+    ./fix-assets-path.patch
+    ./fix-build-docker.patch
+  ];
+  postPatch = ''
+    sed -i 's|@assetsFolder@|'"''${bin}/share/assets"'|g' internal/fileutils/assets.go
+  '';
+
+  preBuild = ''
+     unzip $libFlutter
+     lib=$(mktemp -d)
+     mv libflutter_engine.so $lib/
+     export NIX_LDFLAGS="$NIX_LDFLAGS -L$lib"
+  '';
+
+  postInstall = ''
+    mkdir -p $out/share
+    cp -r assets $out/share/assets
+    chmod -R a+rx $out/share/assets
+
+    wrapProgram "$out/bin/hover" --prefix LD_LIBRARY_PATH : ${stdenv.lib.makeLibraryPath [ glfw libX11 libXcursor libXinerama libXrandr libXxf86vm libXi libGL]}
+  '';
+
+  meta = with stdenv.lib; {
+  preBuildPhases = ["preBuildPhase"];
```

Comment:
```suggestion
```

```
evaluation aborted with the following error message: 'Failed to evaluate hover-unstable-2020-01-23: «unknown-meta»: has an invalid meta attrset:
        - key 'preBuildPhases' is unrecognized; expected one of: 
           ['available', 'badPlatforms', 'branch', 'broken', 'changelog', 'description', 'downloadPage', 'downloadURLRegexp', 'executables', 'homepage', 'hydraPlatforms', 'isBuildPythonPackage', 'isFcitxEngine', 'isGutenprint', 'isIbusEngine', 'knownVulnerabilities', 'license', 'longDescription', 'maintainers', 'name', 'outputsToInstall', 'platforms', 'position', 'priority', 'repositories', 'schedulingPriority', 'tag', 'tests', 'timeout', 'updateWalker', 'version']'
```

Reply:
Sorry, I pushed to check the darwin in CI, is there any way to nix-build for darwin to check if derivation is compilling?

Reply:
And it looks like I moved this `preBuildPhase` accidentally :man_facepalming: